### PR TITLE
Doc changes to UI deployment and dev container getting started guides

### PR DESCRIPTION
**Pull request by bspender**
# Description
Doc changes only - added some additional formatting, links, and some 101 references for Win10/WSL

## Issue reference

The issue this PR will close: # n/a

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles or validates correctly
* [ ] All tests pass (manual and automated)
* [ ] BASH scripts have been validated using `shellcheck`
* [x] The documentation is updated to cover any new or changed features
* [x] Markdown files have been linted using the recommended linter. (See `.vscode/extensions.json`.)
* [ ] Relevant issues are linked to this PR

### CODE REVIEW

File: src/docs/ui-deployment.md [hunk 1]
```diff
@@ -24,12 +24,31 @@ To get started, you'll need to be running from a bash/zsh environment. If you ar
...
+    
+    export TENANT_ID="<TENANT_ID>"
+    export SUBSCRIPTION_ID="<SUBSCRIPTION_ID>"
+    export TF_ENV="<TERRAFORM_ENVIRONMENT>"
+    export MLZ_ENV="<ENVIRONMENT_NAME>"
+    export LOCATION="<CLOUD_LOCATION>"
+
+    chmod u+x ./scripts/setup_ezdeploy.sh
+    
+    ./scripts/setup_ezdeploy.sh -d build -s $SUBSCRIPTION_ID -t $TENANT_ID -l $LOCATION -e $TF_ENV -m $MLZ_ENV -p 80 -0 $SUBSCRIPTION_ID -1 $SUBSCRIPTION_ID -2 $SUBSCRIPTION_ID -3 $SUBSCRIPTION_I
```
Comment (glennmusa) [inline src/docs/ui-deployment.md:49]: Missing the `D` on the very last argument

```suggestion
    ./scripts/setup_ezdeploy.sh -d build -s $SUBSCRIPTION_ID -t $TENANT_ID -l $LOCATION -e $TF_ENV -m $MLZ_ENV -p 80 -0 $SUBSCRIPTION_ID -1 $SUBSCRIPTION_ID -2 $SUBSCRIPTION_ID -3 $SUBSCRIPTION_ID
```

**Comment by glennmusa** [comment]
Thanks for these updates! Very helpful. I'll leave it to the assigned reviewers who are more familiar to approve.

File: .devcontainer/README.md [hunk 2]
```diff
@@ -12,37 +12,47 @@ All configuration related to the development container is in the `.devcontainer`
...
+- Clone the Mission LZ from GitHub to a local workspace on the host machine. 
+  - **Recommended:** If you are using Windows for the host machine, clone and open this repository on Windows Subsystem for Linux (WSL). WSL is not required, but disk IO performance in the container is better and the overall experience is more consistent when running a development container from WSL.
+    > **NOTE:** When using Windows with WSL as the host machine, we recommend the following additional steps:
+    > - After [installing WSL for Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10), you can run your Linux distribution path right inside the Windows command prompt (or in PowerShell), by entering `wsl.exe` or `bash.exe`. These commands will switch to a display of the Linux command line, using the path for your current directory. This path will appear to be in a mounted folder, `/mnt/c`, because we're now viewing your Windows C:\ drive folder from the Linux subsystem. You can access all of your local computer's file system from within the Linux shell by using this `/mnt/c` mounted file path.
```
Comment (brooke-hamilton) [inline .devcontainer/README.md:18]: Please use the markdown linter defined in the dev container: davidanson.vscode-markdownlint

Comment (brooke-hamilton) [inline .devcontainer/README.md:18]: In addition to running wsl.exe from a command or Powershell window, Windows Terminal is a good option for running WSL. It detects all WSL instances and gives you a separate shell for each one.

File: .devcontainer/README.md [hunk 3]
```diff
@@ -12,37 +12,47 @@ All configuration related to the development container is in the `.devcontainer`
...
 
-1. (Optional) if you'd like to interact with the devcontainer's terminal from another terminal other than VS Code's built in terminal, you can docker's `exec` command.
+1. (*Optional*) If you'd like to interact with the devcontainer's terminal from another terminal other than VS Code's built in terminal, you can use the `docker exec` command.
 
-    > NOTE: VS Code attaches to the container as the user named "vscode", so you have to do the same thing when attaching to a BASH session in the container by specifying the user as an argument to the `docker exec` command. If you do not specify the user then you will be connected as root, which will cause permissions issues in git (if you are launching VS Code from WSL).
+    > **NOTE:** VS Code attaches to the container as the user named "vscode", so you have to do the same thing when attaching to a BASH session in the container by specifying the user as an argument to the `docker exec` command. If you do not specify the user then you will be connected as root, which will cause permissions issues in git (if you are launching VS Code from WSL).
 
     ```BASH
     docker exec --interactive --tty --user vscode missionlz-dev /bin/bash
```
Comment (brooke-hamilton) [inline .devcontainer/README.md:58]: Please fix the last sentence in this file because it is not complete. (This is not related to a change you made.) It should read something like: When you are logged into the dev container's terminal, you will find the working director at `/workspaces/missionlz$`

**Review by brooke-hamilton — CHANGES_REQUESTED** [review]

File: .devcontainer/README.md (hunk 2 above)
Comment (bspender) [reply .devcontainer/README.md:18]: Good call out. Should this be added?

File: .devcontainer/README.md (hunk 3 above)
Comment (bspender) [reply .devcontainer/README.md:58]: I moved this up to the prior bullet (e.g. with the step to "Remote-Containers: Reopen in Container") because it occurs with this step

File: .devcontainer/README.md (hunk 2 above)
Comment (brooke-hamilton) [reply .devcontainer/README.md:18]: I think we can either give them the advice above plus Windows Terminal, or leave out all advice related to how to configure and launch WSL. Either is good I think.

File: src/docs/ui-deployment.md [hunk 4]
```diff
@@ -24,12 +24,31 @@ To get started, you'll need to be running from a bash/zsh environment. If you ar
 
 This process will build the user interface container image on your workstation using Docker, upload the container image to your Azure subscription, and install an instance of the container in Azure Container Instances (ACI). You'll need to have Docker installed locally, as well as the Azure Bash CLI.
 
-From the "src" directory
-
+Log in using the Azure CLI
+```BASH
```
Comment (brooke-hamilton) [inline src/docs/ui-deployment.md:28]: Just a few more linting issues in this file.

**Review by brooke-hamilton — CHANGES_REQUESTED** [review]

File: src/docs/ui-deployment.md (hunk 4 above)
Comment (bspender) [reply src/docs/ui-deployment.md:29]: Thank you. I was abel to fix the issues in the areas I added.

**Review by bspender — COMMENTED** [review]
I wasn't able to "fix" the 1-5 [Ordered List] list near the end of this file on lines 97, 103, 130, and 152 (these were already here, but it showed up when I put the file through the linter). Is that ok?

**Review by brooke-hamilton — APPROVED** [review]
❤️
